### PR TITLE
Better readme

### PR DESCRIPTION
**Pull request by tobbe-j**
Better readme to help new users or developers.

Ideas:
- quick jump on top -- added ToC
- quick jump to top -- added go-to-top link 
- quick jumps to demo (video) -- added link to demo video
- add credit to github users -- add links to all github users of people involved
- add link to landing page -- added link to landing page

## Review Checklist

- [x] All changes are described in the PR description
- [x] New feature and breaking changes have documentation
- [x] New endpoints have API tests
- [x] New frontend features have Enzyme tests
- [x] No unnecessary or debugging code
- [x] Questionable code is clearly marked with comments
- [x] Manually tested with latest Firefox, Safari and Chrome

### CODE REVIEW

**Comment by tobbe-j** [comment]
[[task] [misc] Better github readme, prep for open source Tobias](https://trello.com/c/eJn5hlmv/171-task-misc-better-github-readme-prep-for-open-source-tobias)

File: README.md [hunk 1]
```diff
@@ -37,8 +55,92 @@ docker-compose up --build
...
 [node]: https://nodejs.org/en/
+[golang]: https://golang.org/
+[docker]: https://www.docker.com/
+[docker-compose]: https://docs.docker.com/compose/
+[eslint]: https://eslint.org/
+[airbnb-config]: https://github.com/airbnb/javascript
+[jest]: https://jestjs.io/
+[enzyme]: https://github.com/airbnb/enzyme
+[mocha]: https://mochajs.org/
+[aalto]: https://aalto.fi
```
Comment (HenrikHelenius) [inline README.md]: https://www.aalto.fi/department-of-computer-science

**Review by HenrikHelenius — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -37,8 +55,92 @@ docker-compose up --build
...
+# Then run tests for backend
+cd ../server/
+npm run test:api
+```
+
+The frontend test suite is done with [jest][jest] and [enzyme][enzyme]. The backend tests are currently only for the api-endpoints and are run with [mocha][mocha].
+
+## Credits
+
+This project was started during the course CS-C2130 Software Project at [Aalto University][aalto] for the company [Sievo][sievo].
```
Comment (HenrikHelenius) [inline README.md]: Tag github users

**Review by HenrikHelenius — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (HenrikHelenius) [inline README.md]: The project was supported by Sievo

**Review by HenrikHelenius — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,8 +1,46 @@
-# Feedbacker-Forum
+<img src="docs/images/feedbacker-logo.png" height="50%" width="50%">
```
Comment (HenrikHelenius) [inline README.md]: is margin possible? add negative space under

File: README.md [hunk 4]
```diff
@@ -37,8 +77,114 @@ docker-compose up --build
...
+- [@HenrikHelenius](https://github.com/HenrikHelenius)
+
+## License
+
+The project is licensed under the [MIT][MIT] license, see [LICENSE](LICENSE) for more details.
+
+## Screenshots
+
+#### Demo
+Demo video [Feedbacker-Forum](https://www.youtube.com/watch?v=bkxmN-j-ae0)
```
Comment (HenrikHelenius) [inline README.md]: Fake video embed like this: https://stackoverflow.com/questions/4279611/how-to-embed-a-video-into-github-readme-md

File: README.md [hunk 5]
```diff
@@ -1,8 +1,46 @@
...
+
+[![Build Status](https://travis-ci.org/ktk-2005/Feedbacker-Forum.svg?branch=develop)](https://travis-ci.org/ktk-2005/Feedbacker-Forum) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
+
+Feedbacker-Forum is a modern, open-source automated deployment pipeline solution for developers to help gather feedback easily. In other words the developer can trigger a deployment process that ships a version of the current code for the product and gets a simple “feedback-able“ link in return.
+
+ This link can be shared to peers that the developer craves feedback from.
+The developer can then create surveys, recieve answers and discuss the live UI with peers.
+Feedbacker-Forum also supports creating "feedback-able" links for existing external websites.
+
+You can request access to try out Feedbacker-forum from [Feedbacker.site](https://feedbacker.site/about/)
```
Comment (HenrikHelenius) [inline README.md]: Remove this, just put it like bla bla landing page: /about etc

File: README.md [hunk 6]
```diff
@@ -37,8 +77,114 @@ docker-compose up --build
...
+npm run test:api
+```
+
+The frontend test suite is done with [jest][jest] and [enzyme][enzyme]. The backend tests are currently only for the api-endpoints and are run with [mocha][mocha].
+
+## Credits
+
+This project was started during the course CS-C2130 Software Project at [Aalto University][aalto].
+The project was supported by [Sievo][sievo].
+The project was created by
```
Comment (HenrikHelenius) [inline README.md]: created by:

File: README.md [hunk 7]
```diff
@@ -37,8 +77,114 @@ docker-compose up --build
...
+- [@bqqbarbhg](https://github.com/bqqbarbhg)
```
Comment (bqqbarbhg) [inline README.md]: Why namedropping?

Comment (tobbe-j) [reply README.md]: Considered unnecessary, removed

File: README.md [hunk 8]
```diff
@@ -1,8 +1,49 @@
...
+
+
+
+
+[![Build Status](https://travis-ci.org/ktk-2005/Feedbacker-Forum.svg?branch=develop)](https://travis-ci.org/ktk-2005/Feedbacker-Forum) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
+
+Feedbacker-Forum is a modern, open-source automated deployment pipeline solution for developers to help gather feedback easily. In other words the developer can trigger a deployment process that ships a version of the current code for the product and gets a simple “feedback-able“ link in return.
+
+ This link can be shared to peers that the developer craves feedback from.
+The developer can then create surveys, recieve answers and discuss the live UI with peers.
```
Comment (joonalillfors) [inline README.md]: recieve -> receive

File: README.md [hunk 9]
```diff
@@ -1,8 +1,49 @@
...
+Feedbacker-Forum also supports creating "feedback-able" links for existing external websites [/about](https://feedbacker.site/about/).
```
Comment (joonalillfors) [inline README.md]: what is the meaning of this /about?

File: README.md [hunk 10]
```diff
@@ -1,8 +1,49 @@
...
+  * [Running tests](#Running-tests)
+* [Credits](#Credits)
+* [License](#License)
+* [Screenshots](#Screenshots)
+  * [Demo](#demo)
+
 
 ## Setting up
 
+The project consist of a frontend, backend, proxy and database. The different parts can either be setup manually, or run together using containers.
```
Comment (joonalillfors) [inline README.md]: The project -> the application?

File: README.md [hunk 11]
```diff
@@ -1,8 +1,49 @@
...
+
+### Prerequisites
+
 The project is built using [Node.js][node], currently actively tested with version 10.13.
+The project also requires [go][golang] version 1.11 for running the proxy.
```
Comment (joonalillfors) [inline README.md]: project -> application?

File: README.md [hunk 12]
```diff
@@ -37,8 +80,105 @@ docker-compose up --build
...
 
-## Getting started
+## Usage
 
-For further documentation see [docs/getting-started.md](docs/getting-started.md).
+The project consist of two separate main parts - the dashboard view for the developers, and the feedbacker view for giving feedback.
+
+#### Dashboard
+
+The dashboard view is for developers to add, remove and manage instances.
```
Comment (joonalillfors) [inline README.md]: add -> create?

File: README.md [hunk 13]
```diff
@@ -37,8 +80,105 @@ docker-compose up --build
...
+After creating an instance the developer recieves a simple link that can be shared, so that people easily can view the instance and give feedback.
```
Comment (joonalillfors) [inline README.md]: recieves -> receives

Comment (joonalillfors) [reply README.md]: people can easily view

File: README.md [hunk 14]
```diff
@@ -37,8 +80,105 @@ docker-compose up --build
...
+
+#### Feedbacker
+
+The feedbacker view is where the users can use, view and comment on the UI. It consists of two main parts, the survey panel and the comment panel.
+[Screenshot](#feedbacker-image)
+
+#### Features
+
+A brief list of supported features
+* Automatically deploy your app from git, boh public repos and private Github repos supported
```
Comment (joonalillfors) [inline README.md]: boh -> both, repos or repositorys?

File: README.md [hunk 15]
```diff
@@ -37,8 +80,105 @@ docker-compose up --build
...
+* Alternatively use an already live site, simply submit the url
+* Easily discuss and cultivate ideas with theaded discussion, no blocking signup required
```
Comment (joonalillfors) [inline README.md]: threaded discussion

File: README.md [hunk 16]
```diff
@@ -37,8 +80,105 @@ docker-compose up --build
...
+* Tag specific elements to link your comment to a specific part of the UI
+* Comment anonymously on instances where you don't want to share your identity
+* Specify the feedback recieved by creating surveys with normal or multiple choice questions
```
Comment (joonalillfors) [inline README.md]: received

Comment (joonalillfors) [reply README.md]: free form or multiple choice questions?

File: README.md [hunk 17]
```diff
@@ -37,8 +80,105 @@ docker-compose up --build
...
+* Protect sensitive deployments with a passphrase
+* Boost productivity by integrating Feedbacker-Forum to slack with a SlackBot
```
Comment (joonalillfors) [inline README.md]: Slack

File: README.md [hunk 18]
```diff
@@ -37,8 +80,105 @@ docker-compose up --build
...
+
+
+## Contributing
+
+The project is still in a quite early development stage and is actively being developed.
```
Comment (joonalillfors) [inline README.md]: project -> application?

File: README.md [hunk 19]
```diff
@@ -37,8 +80,105 @@ docker-compose up --build
...
+Changes or additions can be proposed by creating issues or pull requests on github. Running a local development version can be setup as seen in [Setting Up](#Setting-up).
```
Comment (joonalillfors) [inline README.md]: as seen in -> as shown in?

File: README.md [hunk 20]
```diff
@@ -37,8 +80,105 @@ docker-compose up --build
...
+
+### Running tests
+
+The project contains has style checking using [eslint][eslint] with a slightly modified [airbnb-config][airbnb-config].
```
Comment (joonalillfors) [inline README.md]: the project contains has

File: README.md (hunk 9 above)
Comment (joonalillfors) [reply README.md]: it doesn't quite flow with the sentence

File: README.md [hunk 21]
```diff
@@ -1,8 +1,47 @@
...
-The project is built using [Node.js][node], currently actively tested with version 10.13.
+The application consist of a frontend, backend, proxy and database. The different parts can either be setup manually, or run together using containers.
+
+### Prerequisites
+
+The application is built using [Node.js][node], currently actively tested with version 10.13.
+The application also requires [go][golang] version 1.11 for running the proxy.
+Instances are run in containers with [docker][docker].
+
+If the application is run with containers [docker-compose][docker-compose] is also required for orchestrating the containers.
```
Comment (quvide) [inline README.md]: What does this mean? Docker-compose is not required when not running the app backend itself outside of docker

**Review by HenrikHelenius — APPROVED** [review]
🎃🎃🎃🎃🎃🎃🎃🎃🎃

**Pull request merged**